### PR TITLE
pr_6

### PR DESCRIPTION
**Pull request by sns-seb**

### CODE REVIEW

**Comment by sns-seb** [comment]
first analysis of PR

File: src/main/java/org/sonarqube/A.java [hunk 1]
```diff
@@ -3,7 +3,11 @@
 public class A {
   String message = "foo";
 
+  // TODO fix this
   public String foo() {
+    message = null;
```
Comment (sns-seb2) [inline src/main/java/org/sonarqube/A.java]: ###AWAIl-Fh3VENDBUoczin###
org.sonarqube:pr-decoration:src/main/java/org/sonarqube/A.java:BRANCH:pr_6@6
Complete the task associated to this TODO comment.

File: src/main/java/org/sonarqube/A.java [hunk 2]
```diff
@@ -3,7 +3,11 @@
...
+    message.length();
+
     return message;
```
Comment (sns-seb2) [inline src/main/java/org/sonarqube/A.java:11]: ###AWAIl-Fh3VENDBUoczio###
org.sonarqube:pr-decoration:src/main/java/org/sonarqube/A.java:BRANCH:pr_6@9
The return value of "length" must be used.

Comment (sns-seb2) [inline src/main/java/org/sonarqube/A.java:11]: ###AWAIl-Fh3VENDBUoczip###
org.sonarqube:pr-decoration:src/main/java/org/sonarqube/A.java:BRANCH:pr_6@9
A "NullPointerException" could be thrown; "message" is nullable here.

**Review by sns-seb2 — DISMISSED** [review]
SonarQube has detected 3 issues

**Comment by sns-seb** [comment]
2nd analysis, no change in code

**Review by sns-seb2 — DISMISSED** [review]
SonarQube has detected 3 issues

**Review by sns-seb2 — DISMISSED** [review]
SonarQube has detected 3 issues

**Comment by sns-seb** [comment]
scan again, no change to code

**Review by sns-seb2 — DISMISSED** [review]
SonarQube has detected 2 issues

**Review by sns-seb2 — DISMISSED** [review]
SonarQube has detected 1 issue

**Comment by sns-seb** [comment]
1 bug was resolved as won't fix before the previous review
now resolving the last one

**Review by sns-seb2 — DISMISSED** [review]
SonarQube has detected 0 issue

**Comment by sns-seb** [comment]
you guessed it, there is a bug in the last review :)

**Review by sns-seb2 — APPROVED** [review]

**Pull request closed without merging**